config/locales/client.en.yml

```diff
@@ -143,6 +143,7 @@ en:
     every_two_weeks: "every two weeks"
     every_three_days: "every three days"
     max_of_count: "max of {{count}}"
+    alternation: "or"
```

Comment:
Yeah, wasn't sure about the design here.

Any suggestions? Here is where it's used:

![](https://cloud.githubusercontent.com/assets/627891/7668903/048847e4-fc0a-11e4-885f-7d9280da6ae6.png)